randovania/games/am2r/json_data/The Nest.txt

```diff
@@ -464,11 +468,7 @@ Extra - map_name: rm_a6a18
   * Layers: default
   * Open Passage to Glowfly Maze Access/Dock to Glowfly Maze
   > Dock to Depths Shinespark Cave Access
-      All of the following:
-          Can Use Any Bombs
-          Any of the following:
-              Tunnel Climb
-              Morph Ball and Mid-Air Morph (Intermediate)
+      Can Use Any Bombs and Tunnel Climb
```

Comment:
the mid air morph is possible:
![grafik](https://github.com/randovania/randovania/assets/38186597/1f5af93d-8a65-4318-9d15-fddcdfa16adc)

Reply:
reverted. saw the 2-tile high section but didn't notice there was a slope.